spaces/S000171/README.md

```diff
@@ -0,0 +1,10 @@
+---
+uid: S000171
+slug: brian-example
+name: Brian's Example
+refs:
+  - mo: 416331
+    name: Example of an uncountable scattered space with some properties
+---
+
```

Comment:
Seems it would be useful here to at least list the main properties that this space is meant to exemplify (uncountable, first countable, Hausdorff, Lindelöf scattered space from the description in #130), instead of forcing people to go read the mo reference to find out.

Reply:
done, thanks for the suggestion